src/Shared/E2ETesting/BrowserFixture.cs

```diff
@@ -52,9 +57,41 @@ public BrowserFixture()
             }
         }
 
+        public IWebDriver Browser { get; }
+
+        public ILogs Logs { get; }
+
+        public IMessageSink DiagnosticsMessageSink { get; }
+
+        public static bool HostSupportsBrowserAutomation => string.IsNullOrWhiteSpace(Environment.GetEnvironmentVariable("ASPNETCORE_BROWSER_AUTOMATION_DISABLED")) &&
+            (IsAppVeyor || (IsVSTS && RuntimeInformation.OSDescription.Contains("Microsoft Windows")) || OSSupportsEdge());
+
+        private static bool IsAppVeyor =>
+            Environment.GetEnvironmentVariables().Contains("APPVEYOR");
+
+        private static bool IsVSTS =>
+            Environment.GetEnvironmentVariables().Contains("TF_BUILD");
+
+        private static int GetWindowsVersion()
+        {
+            var osDescription = RuntimeInformation.OSDescription;
```

Comment:
Can you use `Environment.OSVersion` here instead?

Reply:
Got rid of this alltogether